idb/src/main/kotlin/io/github/paulgriffith/kindling/idb/metrics/MetricCard.kt

```diff
@@ -0,0 +1,89 @@
+package io.github.paulgriffith.kindling.idb.metrics
+
+import net.miginfocom.swing.MigLayout
+import org.jdesktop.swingx.border.DropShadowBorder
+import org.jfree.chart.ChartPanel
+import java.awt.Font
+import java.text.DecimalFormat
+import java.text.FieldPosition
+import java.text.NumberFormat
+import java.text.SimpleDateFormat
+import javax.swing.JLabel
+import javax.swing.JPanel
+import javax.swing.SwingConstants.CENTER
+
+class MetricCard(val metric: Metric, data: List<MetricData>) : JPanel(MigLayout("fill, ins 10")) {
+    companion object {
+        private val heapFormatter = object : DecimalFormat("0 'MB'") {
+            override fun format(number: Double, toAppendTo: StringBuffer?, pos: FieldPosition?): StringBuffer {
+                return super.format(number / 1_000_000, toAppendTo, pos)
+            }
+        }
+        private val queueFormatter = NumberFormat.getIntegerInstance()
+        private val throughputFormatter = DecimalFormat("0.00 'dp/s'")
+        private val cpuFormatter = NumberFormat.getPercentInstance()
```

Comment:
@paul-griffith This doesn't work since this `NumberFormat` instance's default behavior is to multiply the input values by 100, and in the DB they're already stored as percentage values.

You could do
```
private val cpuFormatter = NumberFormat.getPercentInstance().apply {
    (this as DecimalFormat).multiplier = 1
}
```

But the CPU is still showing as an integer. Not sure if this was intended but I don't mind either way.

Reply:
Yeah, this definitely isn't quite working right. I have to think about a better way to pass in scaling here, or maybe just to the same thing as the megabytes for heap.

Reply:
The cast is safe since that instance's implementation uses DecimalFormat, but I guess that could technically not be true for all JDKs.

Reply:
If that works to fix this and you're testing on Azul, I think it's fine. JDeploy defaults to Azul, I'm pretty sure.